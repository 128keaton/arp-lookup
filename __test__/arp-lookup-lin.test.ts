jest.mock('child_process')

import arp from '../src/index'

describe('On a Linux environment', () => {
  Object.defineProperty(process, 'platform', {
    value: 'linux',
  })
  test('Arp table is parsed', async () => {
    await expect(arp.getTable()).resolves.toMatchSnapshot()
  })
})
