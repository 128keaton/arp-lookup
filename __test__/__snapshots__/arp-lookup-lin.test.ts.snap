// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`On a Linux environment Arp table is parsed 1`] = `
Array [
  Object {
    "ip": "192.168.0.13",
    "mac": "01:00:5e:7f:ff:fa",
    "type": "unknown",
  },
  Object {
    "ip": "192.168.0.3",
    "mac": "ff:ff:ff:ff:ff:ff",
    "type": "unknown",
  },
  Object {
    "ip": "192.168.0.2",
    "mac": "01:00:5e:00:00:fb",
    "type": "unknown",
  },
  Object {
    "ip": "192.168.0.7",
    "mac": "01:00:5e:00:00:16",
    "type": "unknown",
  },
  Object {
    "ip": "192.168.0.6",
    "mac": "01:00:5e:00:00:02",
    "type": "unknown",
  },
  Object {
    "ip": "192.168.0.5",
    "mac": "01:00:5e:7f:ff:fa",
    "type": "unknown",
  },
]
`;
